src/meltano/core/tracking/iglu-client-embedded/schemas/com.meltano/cli_context/jsonschema/1-0-1

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Snowplow context describing a Meltano cli command",
+    "self": {
+        "vendor": "com.meltano",
+        "name": "cli_context",
+        "format": "jsonschema",
+        "version": "1-0-1"
+    },
+    "type": "object",
+    "properties": {
+        "command": {
+            "description": "",
+            "type": "string",
+            "maxLength": 1024
+        },
+        "sub_command": {
+            "description": "",
+            "type": [
+                "string",
+                "null"
+            ],
+            "maxLength": 1024
+        },
+        "option_keys": {
+            "description": "",
+            "type": "array",
+            "items": {
+                "description": "",
+                "type": "string",
+                "maxLength": 256
+            }
+        },
+        "options": {
```

Comment:
@pnadolny13 prefs on how you'd like this new field to look? Here are two examples of how it currently looks:

```json
# from a `meltano --log-level=warning run --dry-run tap-gitlab target-jsonl`
"data": {
                            "command": "run",
                            "sub_command": null,
                            "option_keys": [],
                            "options": {
                                "dry_run": true,
                                "blocks": "e1c3e8486805f40d42c1887b4e8928c417d8d3a998b3e27a3e9b48b1dac4aa7a",
                                "full_refresh": false,
                                "no_state_update": false,
                                "force": false,
                                "database_uri": null,
                                "log_level": "4bd9354bb652233497a0f42647464b81b83a83654fcf1511263a5d0120d74c91",
                                "log_config": null,
                                "verbose": 0,
                                "environment": null,
                                "no_environment": false
                            }
                        }
```

```json
# from a `meltano --log-level=warning schedule list --format=text`
"data": {
                            "command": "schedule",
                            "sub_command": "list",
                            "option_keys": [],
                            "options": {
                                "format": "982d9e3eb996f559e633f4d194def3761d909f5a3b647d1a851fead67c32c9d1",
                                "database_uri": null,
                                "log_level": "4bd9354bb652233497a0f42647464b81b83a83654fcf1511263a5d0120d74c91",
                                "log_config": null,
                                "verbose": 0,
                                "environment": null,
                                "no_environment": false
                            }
                        }
```